common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -667,7 +668,7 @@ class NativeAuthMsalController : BaseNativeAuthController() {
 
         // Convert AcquireTokenNoScopesCommandParameters into SilentTokenCommandParameters,
         // so we can use it in BaseController.getCachedAccountRecord()
-        val silentTokenCommandParameters =
+        var silentTokenCommandParameters =
```

Comment:
Why do we still need this method?

Reply:
We don't need this method now. The method  `acquireTokenSilently` depends on `RefreshCommand` class that is present in common project. I cannot move the method `acquireTokenSilently` to `BaseController` in common4j as that will break our dependency pattern. I will see if we can solve this dependency problem.

Reply:
Got it, my bad. I don't think we should move `acquireTokenSilently` to `BaseController`. Let's not change such fundamental MSAL logic. 

At the very least we could try to have our `NativeAuthController` extend `LocalMsalController`, so we can use that `acquireTokenSilent()`, but I think that won't work (or we would've done it already).